src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2042,6 +2042,7 @@ def aks_agentpool_add(cmd,      # pylint: disable=unused-argument,too-many-local
                       spot_max_price=float('nan'),
                       public_ip_per_vm=False,
                       labels=None,
+                      mode="User",
```

Comment:
So be default the new node pool is "User" mode? Customer can set to "System", right?

Reply:
Yes. New nodepools will default to User and customer can change to System.